developing_SoCCo.py

```diff
@@ -0,0 +1,471 @@
+
```

Comment:
What is the purpose of this file? It looks like a script, but can't be run in one shot, and doesn't seem to do all the imports that are needed. Is it in the process of being replaced by the ipython notebook? If so, I would propose to finish that transition, remove this file, and move the notebook to the examples folder (newly created).